test/jld.jl

```diff
@@ -255,6 +255,14 @@ typ{T}(x::Int64, ::Type{T}) = Base.box(Typ{T}, Base.unbox(Int64,x))
 abstract UnexportedT
 end
 
+# test mmapping of small arrays (Issue #192)
+fid = jldopen(fn, "w", mmaparrays=true)
+write(fid, "a", [1:3])
+assert(ismmappable(fid["a"]))
```

Comment:
This would be better as `@test ismmappable(fid["a"])`, since that will provide a more informative error message on failure.